README.md

```diff
@@ -59,7 +64,7 @@ Lightly requires **Python 3.6+**. We recommend installing Lightly in a **Linux**
 
 - hydra-core>=1.0.0
 - numpy>=1.18.1
-- pytorch_lightning>=0.10.0   
+- pytorch_lightning>=1.0.4 
```

Comment:
There are now more dependencies, checkout out the base.txt file:

```
hydra-core>=1.0.0
numpy>=1.18.1
pytorch_lightning>=1.0.4
requests>=2.23.0
torchvision
tqdm>=4.44
certifi >= 14.05.14
six >= 1.10
python_dateutil >= 2.5.3
setuptools >= 21.0.0
urllib3 >= 1.15.1
```

Reply:
Obviously you don't have to list all of them. E.g. urllib3 is also a dependency of requests.

Reply:
I'm not sure I would include them. I would rather wish to check whether we really need all of them. 
I raised the question already once. I don't see why a lightly user needs setuptools^^
Let's also fix this in this sprint. I'll create a follow-up issue.

Reply:
Follow up issue is here: https://github.com/lightly-ai/lightly/issues/219